db/changes.go

```diff
@@ -774,9 +774,13 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 			}
 
 			if options.Revocations && db.user != nil {
-				channelsToRevoke := db.user.RevokedChannels(options.Since.SafeSequence())
-				for channel, triggeredBy := range channelsToRevoke {
-					feed := db.buildRevokedFeed(db.changeCache.getChannelCache().getSingleChannelCache(channel), options, triggeredBy, to)
+				revocationCheckSeq := options.Since.SafeSequence()
+				if options.Since.TriggeredBy > 0 {
+					revocationCheckSeq = options.Since.TriggeredBy - 1
```

Comment:
As discussed, revocationCheckSeq should be the lower of lowSeq and (triggeredBy - 1).